Assets/Scripts/AI/States/Patrol.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+using Random = UnityEngine.Random;
+
+public class Patrol : StateMachineBehaviour
+{
+    [SerializeField]
```

Comment:
I don't think the [SerializeField] property does anything when attached to a `public` member. Possibly redundant?

Reply:
Yeap removed